src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -99,6 +99,77 @@ class ColumnEncodingSegment(BaseSegment):
     )
 
 
+@redshift_dialect.segment()
+class CompressionSegment(BaseSegment):
+    """Compression segment.
+
+    Indicates file compression.
+
+    As specified by: https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-file-compression.html
+    """
+
+    type = "compression_segment"
```

Comment:
Should we consider calling it `compression_type_segment` or just `compression_type`, since this segments specifies which type of compression to use?